agent/engine/docker_task_engine.go

```diff
@@ -916,9 +920,22 @@ func (engine *DockerTaskEngine) createContainer(task *apitask.Task, container *a
 	// If the container is using a special log driver type "awsfirelens", it means the container wants to use
 	// the firelens container to send logs. In this case, override the log driver type to be fluentd
 	// and specify appropriate tag and fluentd-address, so that the logs are sent to and routed by the firelens container.
-	// For reference - https://docs.docker.com/config/containers/logging/fluentd/.
+	// Update the environment variables FLUENT_HOST and FLUENT_PORT depending on the supported network modes - bridge
+	// and awsvpc. For reference - https://docs.docker.com/config/containers/logging/fluentd/.
 	if hostConfig.LogConfig.Type == logDriverTypeFirelens {
 		hostConfig.LogConfig = getFirelensLogConfig(task, container, hostConfig, engine.cfg)
+		if container.GetNetworkMode() == "" || container.GetNetworkMode() == "bridge" {
+			IPAddress, _ := getBridgeIP(task.GetFirelensContainer().GetNetworkSettings())
```

Comment:
IPAddress -> ipAddress. also i think we should return error if we fail to get the ip here